src/applications/disability-benefits/2346/containers/OrderTable.jsx

```diff
@@ -0,0 +1,186 @@
+import ErrorableCheckbox from '@department-of-veterans-affairs/formation-react/ErrorableCheckbox';
+import React from 'react';
+import { connect } from 'react-redux';
+import { focusElement } from '../../../../platform/utilities/ui';
+import { getDLCData, updateDLCData } from '../actions';
+
+class OrderHistory extends React.Component {
+  // NOTE: Using React state for checkboxes -@maharielrosario at 1/8/2020, 7:47:21 PM
+  // When discussed with other devs, the concensus was to have a stateful
+  // component to house our checkboxes' state. But all other state is in Redux.
+  // Is there an easy way to house individual checkbox state in Redux?
+
```

Comment:
I think you need to ask yourself if other components or parts of the app need to know about the 'checked' state of the box.  If that answer is no, then use local state.  If you need to change something in redux state based on that checkbox, you can still dispatch an action to do that but you may not need to keep its checked state in redux.